packages/front-end/src/app/pages/orgs/edit-user/index.vue

```diff
@@ -105,6 +110,16 @@
           Name: lab.Name,
           LaboratoryId: lab.LaboratoryId,
         };
+      })
+      .sort((labA, labB) => {
+        // Lab Manager labs first
+        if (labA.LabManager && !labB.LabManager) return -1;
+        if (!labA.LabManager && labB.LabManager) return 1;
+        // then Lab Technician
+        if (labA.LabTechnician && !labB.LabTechnician) return -1;
+        if (!labA.LabTechnician && labB.LabTechnician) return 1;
+        // then sort by name
+        return stringSortCompare(labA.Name, labB.Name);
```

Comment:
This is great. I have noticed if you grant access, the list doesn't re-sort until a page refresh (although re-sorting works as expected for the roles changing).

Could you perhaps hook in a list refresh to the Grant Access success event?

![image](https://github.com/user-attachments/assets/902fed93-6914-4416-bb8e-6064850e9e4a)

Reply:
Also this sort function seems the same as used in `packages/front-end/src/app/pages/labs/[labId]/index.vue` - I'd consider abstracting them for re-use to a new sort composable.